### DISCUSSION TITLE
I can't attach PDF files to a reply

### DISCUSSION THREAD

**Discussion by dimar7** [botman/botman#1400] [General]
this is the code:

```
<?php
use BotMan\BotMan\Messages\Outgoing\OutgoingMessage;
use BotMan\BotMan\Messages\Attachments\File;

$botman = resolve('botman');

$botman->hears('Send PDF', function ($bot) {

    $attachment = new File('/pdf/example1.pdf', [
        'custom_payload' => true,
    ]);
    $message = OutgoingMessage::create('This is your PDF')
    ->withAttachment($attachment);
    $bot->reply($message);

});
```
It shows the message "This is your PDF" but without attachment, I tried with the full url, but doesn't work either.  The pdf folder is located in the public folder.

**Comment by filippotoso** [comment]
Which driver are you using?

**Reply by dimar7** [reply]
facebook driver

**Reply by dimar7** [reply]
images work fine

**Comment by filippotoso** [comment]
The first parameter of the File constructor must be a full url:

https://github.com/botman/botman/blob/800f0f89149abdd04ad4aa57b789376256bf1f1b/src/Messages/Attachments/File.php#L20

If still doesn't work, open an issue in the Facebook Driver repository.

**Reply by dimar7** [reply]
It doesn't work, I will open an issue there.
